middlewares/validators/user.js

```diff
@@ -2,9 +2,12 @@ const joi = require('joi')
 
 const createUser = async (req, res, next) => {
   const schema = joi.object().keys({
+    phone_number: joi.string().optional(),
```

Comment:
The phone number field is mentioned as `phone` in the API contract of users. You can have a look at this : 
https://github.com/Real-Dev-Squad/website-api-contracts/tree/main/users#users

So, what do you think should we changes it to `phone` here instead of `phone_number`, or vice-versa?
Thank you! 😃

Reply:
@SagarBajpai , can we change this to `phone` as per the contracts?

Reply:
@SagarBajpai , can we change this to phone as per the contracts?

Reply:
DOne